README.md

```diff
@@ -215,9 +220,7 @@ If you like this project please don't forget to give it a star...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
+> Original design idea by Nelson Sakwa on Behance.
```

Comment:
- [ ] [OPTIONAL] Kindly note that this could be stated in a better form just as you have added the MIT link for the License in a way that the user could easily access the author of the original design. See the example below:

I would like to thank [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))[Nelson Sakwa](https://www.behance.net/sakwadesignstudio)